sdk/tables/azure-data-tables/azure/data/tables/aio/_table_client_async.py

```diff
@@ -67,6 +73,68 @@ def __init__(
         self._client._config.version = kwargs.get('api_version', VERSION)  # pylint: disable = W0212
         self._loop = loop
 
+    @classmethod
+    def from_connection_string(
+            cls, conn_str,  # type: str
+            table_name,  # type: str
+            **kwargs  # type: Any
+    ):
+        # type: (...) -> TableClient
+        """Create TableClient from a Connection String.
+
+        :param conn_str:
+            A connection string to an Azure Storage or Cosmos account.
+        :type conn_str: str
+        :param table_name: The table name.
+        :type table_name: str
+        :returns: A table client.
+        :rtype: ~azure.data.tables.TableClient
+        """
+        account_url, secondary, credential = parse_connection_str(
```

Comment:
Does parse_connection_str() need to be imported?

Reply:
yes, good catch thank you!